woocommerce/woocommerce-gateway-stripe
# Add docker config and setup for local development

Fixes #1175.

#### Changes proposed in this Pull Request:

- Add docker config and setup
- Add Xdebug config using the environment variables in `default.env`

#### Testing instructions:

1. Run `npm run up` to create the containers
2. After the containers are created, run `./bin/docker-setup.sh`
3. Navigate to http://localhost:8082 and check if the site is configured with sample products and Stripe is activated
4. Test Xdebug by listening on port 9000 and mapping `"/var/www/html/wp-content/plugins/woocommerce-gateway-stripe"` to the workspace folder
5. Run `npm run down` to stop the containers
